### DISCUSSION TITLE
Can't send notification on demand.

### DISCUSSION THREAD

**Discussion by LocalHeroPro** [laravel/framework#33406] [General] [answered]
- Laravel Version: 7.17.2
- PHP Version: 7.3.19
- Database Driver & Version:

### Description:
As described right here https://laravel.com/docs/7.x/notifications#on-demand-notifications I try send mail on custom/random e-mail address (not user from database) and get that exception:
```php
Call to undefined method Illuminate\Notifications\Notification::route()
```
### Steps To Reproduce:
```php
use Illuminate\Notifications\Notification;
// ...
Notification::route('mail', 'contact@example.com')->notify(new InvoicePaid($invoice));
```

**Comment by GrahamCampbell** [comment, marked as answer]
```php
use Illuminate\Support\Facades\Notification;
```

**Reply by LocalHeroPro** [reply]
Working as expected.
Support, please update docs!

**Reply by GrahamCampbell** [reply]
The docs are already correct. They show an example without a use statement. This will use the facade automatically.

**Reply by GrahamCampbell** [reply]
The static API is explained at https://laravel.com/docs/7.x/notifications#using-the-notification-facade.

**Reply by GrahamCampbell** [reply]
Maybe https://github.com/laravel/docs/pull/6151 will help?

**Reply by LocalHeroPro** [reply]
Nope. Without provided use from your first post I get
```php
Class 'App\Http\Controllers\Notification' not found
```

**Reply by GrahamCampbell** [reply]
You'd need `\Notification::route(...);`

**Reply by LocalHeroPro** [reply]
True, but without 
```php
use Illuminate\Support\Facades\Notification;
```
that
```php
Notification::route('mail', 'contact@example.com')->notify(new InvoicePaid($invoice));
```
doesn't work.
In my opinion better solution is to add the line what you marked as answer in that thread. Then no one can argue with searching proper 'use line'.
Greeting.
